package/base-files/files/sbin/wifi

```diff
@@ -13,9 +13,6 @@ EOF
 }
 
 ubus_wifi_cmd() {
-	local cmd="$1"
-	local dev="$2"
-
 	json_init
 	[ -n "$2" ] && json_add_string device "$2"
 	ubus call network.wireless "$1" "$(json_dump)"
```

Comment:
Instead of remove the variable wonder if a better idea is to use them in the function? Should improve readability